www_src/main.less

```diff
@@ -94,3 +94,5 @@ body {
     width: 50px;
   }
 }
+
+@import "positionable";
```

Comment:
in this case, no - something clobbers my rules by not namespacing, but I'll happily figure out what as a followup